test/helper_modules.rb

```diff
@@ -48,11 +63,13 @@ def self.load_test_categories_and_transaction_types_to_db(community)
 
     def self.load_categories_and_transaction_types_to_db(community, transaction_types, categories)
       # Load transaction types
-      transaction_types.each do |type, translations|
+      transaction_types.each do |type, opts|
 
         transaction_type = Object.const_get(type.to_s).create!(:type => type, :community_id => community.id)
+        transaction_type.create_transaction_process(process: opts[:process])
+        transaction_type.save!
```

Comment:
You mean `save!`?

The relation between TransactionType and TransactionProcess is so that TransactionType _belongs\_to_ TransactionProcess, which means that the foreign key is store to TransactionType.transaction\_process\_id

When `transaction_type.create_transaction_process` is called, a new TransactionProcess is instantiated AND saved to database. The id of the newly created TransactionProcess is set to TransactionType but NOT saved. That's why `save` is needed.

This wouldn't be the case if the relationship was TransactionType _has\_one_ TransactionProcess. The calling `transaction_type.create_transaction_process` would instantiate and save the TransactionProcess, including the foreign key `transaction_type_id`